subprojects/docs/src/docs/release/notes.md

```diff
@@ -256,6 +256,46 @@ You can now use the `export-keys` flag to export all already trusted keys:
 
 For more details, see the [user manual](userguide/dependency_verification.html#sec:local-keyring).
 
+#### Nested composite builds work in more cases
+
+It is common that builds have an included `build-logic` build for build logic.
+That caused problems when using such build as an included build.
+
+For example, take the following nested included build:
```

Comment:
💭 
Can you say this only using the term build 3 times? Using it so often feels heavy. I can't think of a way though.

Reply:
How about now?

Reply:
I find it much better, I like that we state what is now possible, give an example, indicate what would fail before and then share more details.

The only missing bit for me are links:
* To the [upgrade guide](https://docs.gradle.org/current/userguide/upgrading_version_7.html#changes_to_paths_of_included_builds) for potential breaking change given the new setup
* To general docs on [included builds](https://docs.gradle.org/current/userguide/composite_builds.html) and [`build-logic`](https://docs.gradle.org/current/userguide/sharing_build_logic_between_subprojects.html#sec:convention_plugins)

Reply:
I added some links, PTAL!